.github/workflows/qa.yml

```diff
@@ -0,0 +1,44 @@
+name: QA
+
+on:
+    pull_request:
+        branches:
+            - release/*
+
+jobs:
+    backend:
+        runs-on: ubuntu-18.04
+        steps:
+            -   uses: actions/checkout@v2
+
+            # https://github.com/php-actions/composer
+            -   uses: actions/cache@v1
+                with:
+                    path: vendor
+                    key: ${{ runner.OS }}-build-${{ hashFiles('**/composer.lock') }}
+
+            # https://github.com/shivammathur/setup-php
+            -   uses: shivammathur/setup-php@v1
+                with:
+                    php-version: '7.2'
```

Comment:
Waarom specifiek deze versie?

Reply:
Klopt, het zou ook in andere '^7' versies kunnen.  Maar we moeten er hier toch eentje kiezen zover ik weet om de QA in te draaien... deze heb ik toevallig gekozen.  Heb je een voorstel?

Reply:
Dan zou ik 'm in sync houden met de composer.json, alle code die in deze bundle komt moet geschikt zijn voor 7.0 of hoger, en dus ook hier valideren. Mijn voorstel zou dan zijn `7.0`

Reply:
Heb 'm alvast approved

Reply:
https://github.com/zicht/admin-bundle/pull/50

`7.0` doorgevoerd in `release/5.x` en hierbij gemerged

Reply:
Ok... we hebben ook de dependency `"symfony/console": "^4.4",`.  Deze eist `php ^7.1`.  Dus ik zet deze nu op `7.1`...

Reply:
Ok... zelfde verhaal: `"sonata-project/doctrine-orm-admin-bundle": "^3.15"` eist `php ^7.2`.  En dat zijn we weer waar ik 27 dagen geleden was gebleven :cry: